configlib/structs.go

```diff
@@ -48,7 +48,7 @@ type PkgrConfig struct {
 	Suggests       bool                `yaml:"Suggests,omitempty"`
 	NoRecommended  bool                `yaml:"NoRecommended",omitempty"`
 	Repos          []map[string]string `yaml:"Repos,omitempty"`
-	Rollback       bool                `yaml:"Rollback,omitempty"`
+	NoRollback     bool                `yaml:"no_rollback,omitempty" mapstructure:"no_rollback,omitempty"`
```

Comment:
The underscore doesn't follow the convention we've established for things like "NoSecure" and "NoRecommended"

Reply:
so we're in a "bad" spot around naming - newer stuff has used snakecase, older stuff camelcase. I stared at it a while myself between NoRollback and just went crap. One thing we should potentially look at is aliases - eg just allow either to be used.